File: ts/iq/fb-multiway~get~multiway.ts
/** <iq type="get" xmlns="fb:multiway">. Replies with one of: Success, Error. */
export function buildMultiwaydMultiway(flowId?: string, binaryVersion: number, conferenceName?: string, serverInfoData?: string, transactionId?: number, messageType?: string): BinaryNode {
  return { tag: 'iq', attrs: { to: 's.whatsapp.net', xmlns: 'fb:multiway', type: 'get' }, content: [
      { tag: 'multiway', attrs: { flow_id: flowId, binary_version: binaryVersion, conference_name: conferenceName, server_info_data: serverInfoData, transaction_id: transactionId, message_type: messageType } },
    ] }
}

File: go/iq/fb-multiway~get~multiway.go
// BuildMultiwaydMultiway builds <iq type="get" xmlns="fb:multiway">. Replies with one of: Success, Error.
func BuildMultiwaydMultiway(flowId string, binaryVersion int, conferenceName string, serverInfoData string, transactionId int, messageType string) waBinary.Node {
	return waBinary.Node{
		Tag: "iq",
		Attrs: waBinary.Attrs{"to": "s.whatsapp.net", "xmlns": "fb:multiway", "type": "get"},
		Content: []waBinary.Node{
			{Tag: "multiway", Attrs: waBinary.Attrs{"flow_id": flowId, "binary_version": binaryVersion, "conference_name": conferenceName, "server_info_data": serverInfoData, "transaction_id": transactionId, "message_type": messageType}},
		},
	}
}